channels_fields/serializers.py

```diff
@@ -102,14 +102,14 @@ class FieldChannelSerializer(ChannelAppearanceMixin, serializers.ModelSerializer
     """Serializer for FieldChannel"""
 
     lists = serializers.SerializerMethodField()
-    featured_list = UserListSerializer(many=False, read_only=True)
+    featured_list = LearningResourceSerializer(many=False, read_only=True)
```

Comment:
For the field responses, we return entire LearningResource objects.

@mbertrand What do you think of returning just the IDs?
- **Pros:** No N+1 queries to optimize, simpler implementation, easier to get docs accurate. Also makes some caching issues on frontend easier.
- **Cons:** Less detail immediately available in the API response. *IMO, this isn't a big deal. If you're building a UI around `featured_list`, you probably want the list items, too. And the `/learningpaths/<id>` and `/learningpaths/<id>/resources` calls can be made in parallel.

If this change seems reasonable, I could make it in this PR or in a followup.

Reply:
I think it would be okay. The channels_fields API exists primarily to deliver data to/from the front end anyway, so whatever makes things easier there is fine by me.  

Not particularly important, but I wonder if the app name `channel_fields` should be changed to just `channels` now, since we have gotten rid of the reddit-backed discussion channels.

Reply:
(or changed to some other name entirely)

Reply:
> Not particularly important, but I wonder if the app name channel_fields should be changed to just channels now, since we have gotten rid of the reddit-backed discussion channels.
>
> (or changed to some other name entirely)

How about just "fields"?

I was thinking we should also (1) move the BaseChannel model to the `channels_fields` app, and (2) delete the `discussions` app. How about a followup PR to make all three of those changes?

Reply:
👍  on a follow-up PR and moving the base channels model to the same app.  "fields" seems maybe a little too short/generic for an app name, makes me think of serializer/database fields, maybe Ferdi can weigh in on that.